etc/ci/buildbot_steps.yml

```diff
@@ -97,6 +97,7 @@ arm64:
   - bash ./etc/ci/manifest_changed.sh
 
 windows-dev:
+  - echo $PATH
```

Comment:
maybe add ` -c set` instead, so it will be completed to `bash -l -c set` which should work.